frontend/src/components/common/authorized-menu/styles.module.scss

```diff
@@ -4,9 +4,24 @@
   display: flex;
   flex-direction: column;
   min-width: 290px;
-  height: 89.6vh;
+  height: calc(100vh - 96px);
   padding: 40px;
+  overflow-y: scroll;
   background-color: var(--background-gray-300);
+  scrollbar-width: thin;
+  scrollbar-color: var(--background-gray-200) var(--background-gray-300);
```

Comment:
```suggestion
```
don't want to use this one. only `webkit` prefixed

Reply:
but then it won't work same in firefox